MergerService/src/run.cs

```diff
@@ -59,8 +62,15 @@ public class Run : IRun
             this._gpkgPath = this._configurationManager.GetConfiguration("GENERAL", "gpkgPath");
             this._filePath = this._configurationManager.GetConfiguration("GENERAL", "filePath");
             this._shouldValidate = this._configurationManager.GetConfiguration<bool>("GENERAL", "validate");
-            this._batchSize = this._configurationManager.GetConfiguration<int>("GENERAL", "batchSize");
             this._maxTaskRetriesAttempts = this._configurationManager.GetConfiguration<int>("TASK", "maxAttempts");
+            this._batchMaxSize = this._configurationManager.GetConfiguration<int>("GENERAL", "batchSize", "batchMaxSize");
+            this._batchMaxBytes = this._configurationManager.GetConfiguration<long>("GENERAL", "batchSize", "batchMaxBytes");
+            this._limitBatchSize = this._configurationManager.GetConfiguration<bool>("GENERAL", "batchSize", "limitBatchSize");
+
+            if (!this._limitBatchSize)
```

Comment:
if (!this._limitBatchSize || this._limitBatchSize == 0)

Reply:
done